src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -387,6 +387,16 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help='Name of the destination blob. If the exists, it will be overwritten.')
         c.argument('source_lease_id', arg_group='Copy Source')
 
+    with self.argument_context('storage blob sync') as c:
+        c.extra('destination_container', options_list=['--container', '-c'], required=True,
```

Comment:
`--container` seems ambiguous to me. Does the help text make clear that this is the destination? Why is the destination a URL then?

Reply:
The helper text is clear as 'The sync destination container.'
The validate function is to validate that the url of the blob of destination container with path is valid. The url is to access the destination container with path.